lib/routes/account.js

```diff
@@ -1896,6 +1982,206 @@ module.exports = (
       }
     },
     {
+      method: 'GET',
+      path: '/recovery_emails',
+      config: {
+        auth: {
+          strategy: 'sessionToken'
+        },
+        response: {
+          schema: isA.array().items(
+            isA.object({
+              verified: isA.boolean().required(),
+              isPrimary: isA.boolean().required(),
+              email: validators.email().required()
+            }))
+        }
+      },
+      handler: function (request, reply) {
+        const sessionToken = request.auth.credentials
+        const uid = sessionToken.uid
+
+        if (! features.isSecondaryEmailEnabled()) {
+          return reply(error.featureNotEnabled())
+        }
+
+        log.begin('Account.RecoveryEmailEmails', request)
```

Comment:
We should probably move this above the `if (! features.isSecondaryEmailEnabled()) {`, just in case we need more logs when feature is enabled / disabled.